### ISSUE TITLE
Correct the spelling of litres

### ISSUE THREAD

**Issue by johnjenkman** [samjbmason/punkapi#42]
The recipe data is using the American spelling of **liters** for the boil_volume and volume units.

```javascript
  "volume": {
    "value": 20,
    "unit": "liters"
  },
  "boil_volume": {
    "value": 25,
    "unit": "liters"
  }
```

It should use the international spelling for the measurement which is **litres** to be consistent with the other international spellings used for measurements in the current data set such as grams, kilograms and celsius.

**Comment by sammdec** [comment]
Yeah makes sense

**Comment by sammdec** [comment]
This is fixed with version 2.6.0 now e642b6eaba8e58b0a64a92942e1503853edaf4db
